src/content/docs/en/guides/upgrade-to/v1.mdx

```diff
@@ -1,7 +1,7 @@
 ---
 title: Legacy v0.x Upgrade Guide 
 description: Archived guide documenting changes between pre-v1 versions of Astro
-i18nReady: false
+i18nReady: true
```

Comment:
```suggestion
i18nReady: false
```

The page is outdate. I don't think we need to translate it.

Reply:
Agreed, this page is *really* old.